static/docs/commands-reference/diff.md

```diff
@@ -54,10 +86,50 @@ Git reference.
 
 ```dvc
     $ dvc diff HEAD^
-    dvc diff from 6a819f8fa053f124f6a5487efc824a8c17366c71 to 612f6caf5c5daeb172167db285efd8b169d41b60
+    dvc diff from df613bce6dc0738f71c62d1748e1edfb3b7e4893 to ed10968bcb0dab72563d05712f24ddfff698c87b
 
     diff for 'data/data.xml'
     +data/data.xml with md5 a304afb96060aad90176268345e10355
 
     added file with size 37.9 MB
 ```
+
+## Examples: Specific target accross Git references
+
+We can base this example in the [Experiment Metrics](/doc/get-started/metrics)
+and [Compare Experiments](/doc/get-started/compare-experiments) sections of our
+Get Started guide, which describe different experiments to produce the
+`model.pkl` file.
+> Our sample repository has the `bigrams-experiment` and `baseline-experiment`
+[tags](https://github.com/jorgeorpinel/example-get-started/tags) respectively to
+refernce these experiments.
+
+### Click and expand to setup example
+
+Having followed the previous example's setup, move into the
+**example-get-started** directory. Then make sure that you have the latest code
+and data with the following commands.
+
+```dvc
+    $ git checkout master
+    $ dvc fetch -aT
+```
+
+The `-aT` flag passed to `dvc fetch` makes sure we have all the data files
+related to all existing tags in the repo. You take a look at the available tags
+of our sample repo in https://github.com/jorgeorpinel/example-get-started/tags.
```

Comment:
jorgeorpinel?

Reply:
make `sample repo` a link?

Reply:
Woops, fixed.
And made into link (I thought the doc engine would linkify automatically).

Reply:
Note that this link breaks the 80 column alignment...